src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Classnames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+enum Response {
+  title = 'Title',
+  error = 'Error',
+}
+
+type Props = {
+  addMovie: (movie:Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [search, setSearch] = useState<string>('');
+  const [answear, setAnswear] = useState<Movie | {} | ResponseError>({});
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>(false);
+
+  const onChangeHandel = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearch(event.target.value);
+    if (error) {
+      setError(false);
+    }
+  };
+
+  const requstByMovie = async () => {
+    setIsLoading(true);
+    const response = await getMovie(search);
```

Comment:
use try / catch for this

Reply:
no, because answear if else wrong allways 200 status